### PR TITLE
jgrss/coreg 215

### PR DESCRIPTION
**Pull request by jgrss**
## What is this PR changing?
Addresses #215 and updates installation instructions in the online documentation.

### CODE REVIEW

**Comment by jgrss** [comment]
@mmann1123 In addition to some changes to `setup.cfg` to address #215, take a look at the `install.rst` rearrangement.

**Comment by jgrss** [comment]
Okay, sorry for the pushes after the PR. Should be steady to review now.

File: doc/source/install.rst [hunk 1]
```diff
@@ -457,32 +364,16 @@ Installation Notes
...
-    - `CPLE_OpenFailedError: Unable to open EPSG support file gcs.csv`.
-
-    This error is documented on rasterio's `ReadTheDocs page <https://rasterio.readthedocs.io/en/latest/faq.html>`_ and `GitHub page <https://github.com/mapbox/rasterio/issues/1787>`_. If the suggested solutions do not fix the issue, you can try setting the `GDAL_DATA` environment variable to point to Fiona (which will be installed automatically when installing GeoWombat). For example, if you have setup a virtual environment, the `GDAL_DATA` variable can point to `/path/to/myenv/lib/python3.7/site-packages/fiona/gdal_data`, where `/path/to/myenv` is the name of your virtual environment path directory. Change 3.7 if using a different Python version.
- 
-
 .. note::
 
-    **Install the GDAL Python bindings seperately**
-    GeoWombat will attempt to install the GDAL Python package if the GDAL binaries are installed. However, to install Python GDAL manually, use pip::
+    **EPSG File Missing in Linux:**
```
Comment (mmann1123) [inline doc/source/install.rst:369]: Nice good call

**Review by mmann1123 — APPROVED** [review]
Looks good.

**Pull request merged**